packages/components/forms/src/base-input/BaseInput.styles.ts

```diff
@@ -1,7 +1,26 @@
 import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
+import type { CSSObject } from '@emotion/serialize';
 
-const getStyles = ({ as, isDisabled, isInvalid }) => ({
+const sizeToStyles = ({ size }): CSSObject => {
+  switch (size) {
+    case 'small':
+      return {
+        fontSize: tokens.fontSizeM,
+        lineHeight: tokens.lineHeightM,
+        padding: `${tokens.spacing2Xs} ${tokens.spacingXs}`,
+        height: `32px`,
+      };
+    default:
+      return {
+        fontSize: tokens.fontSizeM,
+        lineHeight: tokens.lineHeightM,
+        height: `40px`,
```

Comment:
This is creating a weird scroll in the textarea component

![Screenshot 2021-09-06 at 14 41 30](https://user-images.githubusercontent.com/6597467/132219204-65834920-68fa-45c9-989f-bc000c398cde.png)

I suggest not applying this height if `as === 'textarea'` or pass this as min-height when base input is textarea